tests/Unit/Patterns/NodeTest.php

```diff
@@ -205,16 +204,6 @@ public function testWithNameAndLabelAndProperties(string $name, string $label, a
         $this->assertSame($variable, $node->getVariable());
     }
 
-    /**
-     * @dataProvider provideBacktickThrowsExceptionData
-     * @param Node $invalidNode
-     */
-    public function testBacktickThrowsException(Node $invalidNode): void
```

Comment:
Using a backtick should still throw an exception, right?

Reply:
Yes, it only accepts alphanumeric characters and underscores. I added an explicit test now